### PR TITLE
Use llvm10 utilities to promote indirect calls

### PR DESCRIPTION
**Pull request by caballa**

### CODE REVIEW

File: include/seahorn/Transforms/Utils/DevirtFunctions.hh [hunk 1]
```diff
@@ -22,6 +22,8 @@ class CompleteCallGraphAnalysis;
 
 namespace seahorn {
 
+//#define USE_BOUNCE_FUNCTIONS
```
Comment (agurfinkel) [inline include/seahorn/Transforms/Utils/DevirtFunctions.hh:25]: you can go all the way and add a cmake option to turn this on during compilation

Comment (caballa) [reply include/seahorn/Transforms/Utils/DevirtFunctions.hh:25]: Yes

**Comment by caballa** [comment]
@agurfinkel : note that we could use as an alternative method to devirtualize the metadata that adds `CalledValuePropagation` which is the LLVM analysis that resolves indirect calls based on the sparse propagator solver. I didn't bother because it shouldn't be better than seadsa.

**Comment by agurfinkel** [comment]
ready to merge

**Comment by caballa** [comment]
Yes, let me move the pragma flag to cmake

**Pull request merged**